### DISCUSSION TITLE
pac auth profile of kind OperatingSystem

### DISCUSSION THREAD

**Discussion by rajyraman** [microsoft/powerplatform-vscode#1054] [Q&A] [answered]
As per this post https://devblogs.microsoft.com/powerplatform/automate-power-platform-environment-creation-solution-imports-with-github-actions-federated-auth-setup/, pac auth create is not required, but somehow pac works in GitHub Actions which is not the case for me.

However locally, pac automatically seems to be creating an auth profile called OperatingSystem

![Image](https://github.com/user-attachments/assets/efcaa5da-e60b-4139-b3a1-ff42c4d1ecb8)

With this automatically created auth profile all pac command seem to work with the -env switch. How is this authentication working? What is pac using to authenticate and create this profile?

**Comment by rajyraman** [comment]
@tehcrashxor - Can you please find the right person to answer this? Thank you.

**Reply by tehcrashxor** [reply]
@petrochuk

**Reply by rajyraman** [reply]
@petrochuk - Any update on this sir?

**Comment by rajyraman** [comment, marked as answer]
Seems like it is driven by this featureflag and works only on Windows. Closing, as I thought I can use in in Az Cloud Shell and GitHub Actions.

![image](https://github.com/user-attachments/assets/d0b229da-4f23-4926-987d-083707c7ba71)
